spec/unit/pdk/util/bundler_spec.rb

```diff
@@ -347,52 +348,30 @@ def bundle_regex
     end
 
     describe '#installed?' do
-      let(:gemfile) { '/Gemfile' }
+      let(:gemfile) { File.join(FIXTURES_DIR, 'basic_module_gemfile') }
 
       before(:each) do
         allow(instance).to receive(:gemfile).and_return(gemfile)
       end
 
-      it 'invokes `bundle check`' do
-        expect_command([bundle_regex, 'check', "--gemfile=#{gemfile}", '--dry-run'], exit_code: 0)
-
-        instance.installed?
-      end
-
-      it 'returns true if `bundle check` exits zero' do
-        allow_command([bundle_regex, 'check', "--gemfile=#{gemfile}", '--dry-run'], exit_code: 0)
-
+      it 'returns true' do
         expect(instance.installed?).to be true
       end
```

Comment:
Maybe wrap this expectation in a context that says "when all gems present" just so the full test description is more clear.